pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories.md

```diff
@@ -0,0 +1,198 @@
+---
+$title: Embed stories in web pages
+$order: 3
+description: 'The AMP Story Player enables you to embed a story that users are able to tap or click through, inside of a web page. Follow this step-by-step guide to learn how.'
+formats:
+  - websites
+  - stories 
+---
+
+Stories are a full-screen immersive experience. They’re hosted on the open web with their own URL, making them easily shareable. But, what if you want to integrate stories into your own site, for example, within a blog, product description or news article? 
+
+The [AMP Story Player](https://github.com/ampproject/amphtml/blob/master/spec/amp-story-player.md) enables you to embed a story that users are able to tap or click through, inside of a web page. Follow this step-by-step guide to learn how.
+
+# Display stories in a non-AMP page
+
+You can embed AMP stories inside a non-AMP page, allowing users to tap or click through the experience without leaving the host document!
+
+[example preview="top-frame" playground="false"]
+```html
+<!doctype html>
+    <head>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script
+        async
+        src="https://cdn.ampproject.org/amp-story-player-v0.js"
+    ></script>
+    <link
+        href="https://cdn.ampproject.org/amp-story-player-v0.css"
+        rel="stylesheet"
+        type="text/css"
+    />
+    <style>
+        header {
+        height: 8vh;
+        width: 100vw;
+        display: flex;
+        justify-content: space-around;
+        color: #545454;
+        background-color: #DDB556;
+        margin-bottom: 3vh;
+        }
+        .content {
+        display: flex;
+        justify-content: space-around;
+        width: 100vw;
+        }
+    </style>
+    </head>
+    <body>
+    <header>
+        <h1>
+        Page Header
+        </h1>
+    </header>
+    <h1>
+        Article Title
+    </h1>
+    <p>
+        Doggo ipsum smol wow very biscit length boy, doing me a frighten.  Borking doggo doggo heckin dat tungg tho, heckin good boys. Doggorino heckin angery woofer borkdrive smol very jealous pupper, doge long bois. Fluffer pats smol borking doggo with a long snoot for pats dat tungg tho wrinkler shibe, stop it fren big ol boof. Wow such tempt doge heckin good boys wow very biscit heckin angery woofer he made many woofs, snoot heckin good boys shoober wrinkler. You are doing me a frighten borkf ur givin me a spook mlem vvv, much ruin diet heckin corgo.
+    </p>
+    <div class="content">
```

Comment:
I find it weird to have a `content` div which only includes the story. Why are you not targeting the amp-story-player element directly?